ssl/t1_trce.c

```diff
@@ -1341,6 +1341,9 @@ static int ssl_print_ticket(BIO *bio, int indent, SSL *s,
         msg += 4;
         BIO_indent(bio, indent + 2, 80);
         BIO_printf(bio, "ticket_age_add=%u\n", ticket_age_add);
+        if (!ssl_print_hexbuf(bio, indent + 2, "ticket_nonce", 1, &msg,
+                              &msglen))
```

Comment:
Does this print the length octet as well?
(If we do not print the length octet we should probably sanity check against msglen.)